pkg/server/testdata/combined_statement_stats

```diff
@@ -0,0 +1,136 @@
+query sort=PCT_RUNTIME limit=100 start=1 end=2 start=1 end=2
+----
+--WHERE--
+ WHERE app_name NOT LIKE '$ internal%' AND app_name NOT LIKE '$$ %' AND aggregated_ts >= $1 AND aggregated_ts <= $2
+--ORDER AND LIMIT--
+ ORDER BY ((statistics -> 'statistics' -> 'svcLat' ->> 'mean')::FLOAT *
+                         (statistics -> 'statistics' ->> 'cnt')::FLOAT) DESC LIMIT $3
+--ARGS--
+[1970-01-01 00:00:01 +0000 UTC 1970-01-01 00:00:02 +0000 UTC 100]
+
+query sort=SERVICE_LAT limit=100 start=1 end=2
```

Comment:
nit: Could we have some test cases where limit is something other than 100 / when it's omitted? Just to verify it's putting the correct value.

Reply:
done.